Add Jupyter notebook examples for using evo-sdk-common, evo-files (#42)
SeequentEvo/evo-python-sdk: packages/evo-sdk-common/README.md

pip install evo-sdk-common
```

## Usage

> See the [Getting Started](docs/quickstart.md) guide for how to use this library. There is also more detailed information about authentication options in [the OAuth examples](docs/oauth.md), and a selection of interactive Jupyter notebooks in the [examples folder](docs/examples).

Reviewer: Was the absolute linking something we could try out here....or leave it to another ticket? Because the quickstart and the oauth.md links are not accessible in the PyPi readme

Author: No, we should write our documentation with standard markdown syntax. PyPI's decision not to transform links in content they're rendering is something that needs to be handled in a unique way to support PyPI - they suggest pre-processing the documentation files and have [some examples on how to do it](https://github.com/pypa/packaging-problems/issues/752#issuecomment-2083428900):
> It's easy with hatchling, you can use hatch-fancy-pypi-readme and statically defined regular expressions. With setuptools, you can open the file in setup.py and process it yourself.

> See the [Getting Started](docs/quickstart.md) guide for how to use this library. There is also more detailed information about authentication options in [the OAuth examples](docs/oauth.md), and a selection of interactive Jupyter notebooks in the [examples folder](docs/examples).

Reviewer 2: ```suggestion
See the [Getting Started](docs/quickstart.md) guide to learn how to use this library. There is more detailed information about authentication options in the [OAuth examples](docs/oauth.md), and a selection of interactive Jupyter notebooks in the [examples folder](docs/examples).
```
